modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -138,6 +139,8 @@ An additional role is required if the installation program is to create a vSpher
 `VirtualMachine.Inventory.CreateFromExisting`
 `VirtualMachine.Inventory.Delete`
 `VirtualMachine.Provisioning.Clone`
+`VirtualMachine.Provisioning.DeployTemplate`
+`VirtualMachine.Provisioning.MarkAsTemplate`
```

Comment:
Both permissions only need to be added on 4.8+ since ova images will be convert to template from ocp4.8 once it is uploaded to vsphere.

Reply:
Lines 142 and 143 removed.

Reply:
I can still see 2 lines in doc from direct link